_data/communication.yml

```diff
@@ -22,9 +22,9 @@ websites:
 
     - name: Customer.io
       url: https://customer.io
-      twitter: customerio
       img: customerio.png
-      tfa: No
+      tfa: Yes
+      software: Yes
 
```

Comment:
The doc URL given in the previous issue should be added.
`doc: https://customer.io/docs/two-factor-auth.html`